docs/docs/docs/developer-resources/testing.md

```
@@ -503,3 +509,81 @@
   }
 }
 ```
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)